tests/ast/test_ast_driver.py

```diff
@@ -110,6 +110,39 @@ def run_test(session, test_source):
     schema=['num', 'Owner\\'s""opinion.s']
 )
 mock.write.save_as_table("\\"the#qui.ck#bro.wn#\\"\\"Fox\\"\\"won\\'t#jump!\\"")
+
+# Set up data used for union, intersection, and except tests.
```

Comment:
This list may not age super well, I'd just say something more vague, without referencing specific tests.

Reply:
That's a good point, I could say "set operators" instead.